src/file_browser/mod.rs

```diff
@@ -210,8 +211,12 @@ impl FileBrowserWidget {
 
         let store = &self.store;
         let state_ref = &self.state;
-        self.tree
-            .connect_test_expand_row(clone!(store, state_ref => move |_, iter, _| {
+        self.tree.connect_test_expand_row(glib::clone!(
+            #[strong]
```

Comment:
What does the `strong` attribute does? Couldn't find it in the docs.

Reply:
Same as `@strong` before, and what the custom `clone!` macro here did previously without any attributes: It calls `store.clone()` and moves that into the closure.

See https://gtk-rs.org/gtk-rs-core/stable/latest/docs/glib/macro.clone.html#passing-a-strong-reference

Reply:
LGTM